components/x-audio/src/components/__tests__/Players.test.jsx

```diff
@@ -0,0 +1,99 @@
+const { h } = require('@financial-times/x-engine');
+const { mount } = require('@financial-times/x-test-utils/enzyme');
+const { ExpandedPlayer, MinimisedPlayer } = require('../Players');
+const {
+	getProps,
+	playPauseButtonSelector,
+	closeButtonSelector,
+	forwardButtonSelector,
+	rewindButtonSelector,
+	minimisedButtonSelector,
+	playBackRateButtonSelector,
+	imageSelector,
+	titleSelector,
+	seriesNameSelector,
+	timelineSelector
+} = require('../../../__tests__/test-helper');
+
+jest.mock('../classnames-helper', () => {
+	return (...classes) => classes.join(' ');
+});
```

Comment:
Feels like this mock is being used in most of the test files now, maybe worth extracting to a helper?

Reply:
👍